packages/core/talos/Makefile

```
@@ -0,0 +1,38 @@
+NAME=talos
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)